guide/additional-info/changes-in-v12.md

```diff
@@ -1977,43 +1977,43 @@ Both the `receiver.createOpusStream()` and `receiver.createPCMStream()` methods
 
 #### VoiceReceiver#destroy
 
-This method has been removed entirely, refer to `StreamDispatcher#destroy` for documentation.
+This method was removed entirely; refer to `StreamDispatcher#destroy` for documentation.
 
 #### VoiceReceiver#destroyed
 
-This property has been removed entirely.
+This property was removed entirely.
 
 #### VoiceReceiver#opus
 
-This event has been removed entirely.
+This event was removed entirely.
 
 #### VoiceReceiver#pcm
 
-This event has been removed entirely.
+This event was removed entirely.
 
 #### VoiceReceiver#recreate
 
-This method has been removed entirely.
+This method was removed entirely.
 
 #### VoiceReceiver#voiceConnection
 
-This property has been removed entirely.
+This property was removed entirely.
 
 #### VoiceReceiver#warn
 
-This event has been removed entirely, use the `receiver.debug` event instead.
+This event was removed entirely; use the `receiver.debug` event instead.
 
 ### VoiceRegion
 
 #### VoiceRegion#sampleHostname
 
-This property has been removed entirely.
+This property was removed entirely.
 
 ### Webhook
 
 #### Webhook#avatarURL
 
-`webhook.avatarURL` is now a method, as opposed to a property. It also allows you to determine the file format and size to return. If the `dynamic` option is provided you will receive a `.gif` URL if the image is animated, otherwise it will fall back to the specified `format` or its default `.webp` if none is provided.
+`webhook.avatarURL` is now a method, as opposed to a property. It also allows you to determine the file format and size to return. If the `dynamic` option is provided, you will receive a `.gif` URL if the image is animated; otherwise, it will fall back to the specified `format` or its default `.webp` if none is provided.
```

Comment:
```suggestion
`webhook.avatarURL` is now a method, as opposed to a property. It also allows you to determine the file format and size to return. If you provide the `dynamic` option, you will receive a `.gif` URL if the image is animated; otherwise, it will fall back to the specified `format` or its default `.webp`.
```